src/Utilities/Helper.php

```diff
@@ -151,10 +151,12 @@ public static function getOrMethod($method)
      * Converts array object values to associative array.
      *
      * @param mixed $row
+     * @param array $filters
      * @return array
      */
-    public static function convertToArray($row)
+    public static function convertToArray($row, $filters = [])
     {
+        $row  = method_exists($row, 'makeHidden') ? $row->makeHidden(array_get($filters, 'hidden', [])) : $row;
```

Comment:
Maybe we should avoid array functions due to deprecation like on your other PR? 

Anyways, I think there are other array functions in the package that needs to be updated maybe on another PR. A minor issue and this is still good for now.